Modules/xASL_module_Population.m

```diff
@@ -235,16 +235,6 @@
     % Iterate over DataTypes
     for iDataType=1:length(x.S.DataTypes)
         x.S.InputDataStr = x.S.DataTypes{iDataType};
-        

```

Comment:
This is still valid, did you move this elsewhere?

Reply:
Yes of course - this is now inside the GetRoiStatistics functions